src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int START_INDEX = 0;
+    private static final int SIX_FIGURES = 6;
```

Comment:
Naming should be based on purpose, not value itself
```suggestion
    private static final int FIGURES_AMOUNT = 6;
```